cargo/cargo_build_script.bzl

```diff
@@ -187,7 +204,8 @@ _build_script_run = rule(
             doc = "The name of the native library this crate links against.",
         ),
         "deps": attr.label_list(
-            doc = "The dependencies of the crate defined by `crate_name`",
+            doc = "The Rust dependencies of the crate defined by `crate_name`",
+            providers = [DepInfo],
```

Comment:
After doing a bit more tinkering, I also think this change should be added. The only targets passed to `deps` that end up being used are ones with `DepInfo` so I thought by adding this requirement, it'd be clearer to users that they should be using `data` if they want to add dependencies here.

Reply:
Good thinking!